components/search/MoreFacetsDropdownOptions.vue

```diff
@@ -8,6 +8,7 @@
     <b-form-checkbox-group
       class="option-group"
       plain
+      @change="selected"
```

Comment:
while this could be okay for now to demonstrate the selection working, this will need to be refactored to only update when the 'apply' button is clicked, not when an option is selected.